benchmarks/benchmarks.py

```diff
@@ -25,14 +26,25 @@
 from mitiq.interface import mitiq_cirq
 
 
+if os.environ.get("BENCHMARK_CI"):
+    params = {"nqubits": [1], "depth": [1, 2, 3], "num_samples": [10]}
+else:
+    params = {"nqubits": [2], "depth": [2, 4, 6], "num_samples": [20]}
```

Comment:
I think `num_samples = 100` is better. Good to call this `num_pec_samples` or similar to be clear.

Reply:
Might need to have the `timeout` (currently 300 seconds) be larger for these tests.

Reply:
The benchmarks with increased parameters only took 115 seconds on my computer (2020 mbp), so I think 300 should be fine for github actions.